flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/types_manager/config/config1.json

```diff
@@ -0,0 +1,144 @@
+{
+  "projectPath": "test_pack1",
+  "description": "Get all types associated with the package",
+  "types": [
+    {
+      "name": "Address",
+      "editable": true,
+      "metadata": {
+        "label": "Address",
+        "description": ""
+      },
+      "codedata": {
+        "lineRange": {
+          "fileName": "types.bal",
+          "startLine": {
+            "line": 2,
+            "offset": 5
+          },
+          "endLine": {
+            "line": 2,
+            "offset": 12
+          }
+        },
+        "kind": "RECORD"
+      },
+      "properties": {
+        "name": {
+          "metadata": {
+            "label": "Type name",
+            "description": "Unique name to identify the type"
+          },
+          "valueType": "IDENTIFIER",
+          "value": "Address",
+          "optional": false,
+          "editable": true,
+          "advanced": false
+        },
+        "description": {
+          "metadata": {
+            "label": "Type name",
+            "description": "Unique name to identify the type"
```

Comment:
All the properties have the same metadata information

Reply:
Fixed. Thanks!